code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -0,0 +1,199 @@
+	gain_text = "Owl is the keeper of things that quite not are in practice, but in theory are."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_ghoul,/datum/eldritch_knowledge/cold_snap)
+	result_atoms = list(/obj/item/clothing/suit/hooded/cultrobes/void)
+	required_atoms = list(/obj/item/shard,/obj/item/clothing/suit,/obj/item/bedsheet)
+
+/datum/eldritch_knowledge/void_mark
+	name = "Mark of Void"
+	gain_text = "I saw them, the marked ones. The screams.. the silence."
+	desc = "Your mansus grasp now applies mark of flesh status effect. To proc the mark, use your sickly blade on the marked. Mark of flesh when procced causeds additional bleeding."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/spell/void_phase)
+	banned_knowledge = list(/datum/eldritch_knowledge/rust_mark,/datum/eldritch_knowledge/ash_mark,/datum/eldritch_knowledge/flesh_mark)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/void_mark/on_mansus_grasp(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(isliving(target))
+		. = TRUE
+		var/mob/living/living_target = target
+		living_target.apply_status_effect(/datum/status_effect/eldritch/void)
+
+/datum/eldritch_knowledge/spell/void_phase
+	name = "Void Phase"
+	gain_text = "Reality bends under the power of memory, for all is fleeting, and what else stays?"
+	desc = "You gain a long range pointed blink that allows you to instantly teleport to your location, it causes aoe damage around you and your chosen location."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/pointed/void_blink
+	next_knowledge = list(/datum/eldritch_knowledge/rune_carver,/datum/eldritch_knowledge/crucible,/datum/eldritch_knowledge/void_blade_upgrade)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/rune_carver
+	name = "Carving Knife"
+	gain_text = "Etched, carved... eternal. I can carve the monolith and evoke their powers!"
+	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them, to create the carving knife transmute a knife with a glass shard and a piece of paper."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/spell/void_phase,/datum/eldritch_knowledge/summon/raw_prophet)
+	required_atoms = list(/obj/item/kitchen/knife,/obj/item/shard,/obj/item/paper)
+	result_atoms = list(/obj/item/melee/rune_knife)
+
+/datum/eldritch_knowledge/crucible
+	name = "Mawed Crucible"
+	gain_text = "This is pure agony, i wasn't able to summon the dereliction of the emperor, but i stumbled upon a diffrent recipe..."
+	desc = "Allows you to create a mawed crucible, eldritch structure that allows you to create potions of various effects, to do so transmute a table with a watertank"
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/spell/void_phase,/datum/eldritch_knowledge/spell/area_conversion)
+	required_atoms = list(/obj/structure/reagent_dispensers/watertank,/obj/structure/table)
```

Comment:
uh, won't this conflict with the recipes for eldritch water and eldritch armor, respectively?

Reply:
EDIT: Oh, never mind, it won't now. Still, this seems like a rite that's rather easy to accidentally perform while trying to perform a different rite...